sdk/python/kfp/utils.py

```diff
@@ -0,0 +1,61 @@
+def use_secret(secret_name=None, volume_name=None, secret_volume_mount_path=None, env_variable=None, secret_file_path_in_volume=None):
+    """    
+       An operator that configures the container to use a secret.
+       
+       This assumes that the secret is created and availabel in the k8s cluster.
+    
+    Keyword Arguments:
+        secret_name {String} -- The k8s secret name (default: {None})
```

Comment:
nit: also consider using type hints in the function signature.

Reply:
Also explicitly mark which argument is required vs. which is optional.

Reply:
Will update with this.